internal/postprocessor/main.go

```diff
@@ -162,28 +166,41 @@ func (c *config) run(ctx context.Context) error {
 	if _, err := c.Manifest(generator.MicrogenGapicConfigs); err != nil {
 		return err
 	}
-	// TODO(codyoss): In the future we may want to make it possible to be able
-	// to run this locally with a user defined remote branch.
-	if err := c.AmendPRDescription(ctx); err != nil {
-		return err
+	return nil
+}
+
+func (c *config) getDirs() []string {
+	dirs := []string{}
+	for _, module := range c.modules {
+		dirs = append(dirs, filepath.Join(c.googleCloudDir, module))
+	}
+
+}
+
+func (c *config) TidyAffectedMods() error {
+	dirs := c.getDirs()
+	for _, dir := range dirs {
+		if err := gocmd.ModTidy(dir); err != nil {
```

Comment:
This works nice for running locally, but will dirs be populated in the case of owlbot PRs? Maybe we need to find a way to programitically figure this out with git commands.

Reply:
I am resolving this for now, and may look into the possibility of extracting the changed modules for this part of the postprocessing with git commands in another PR.